pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -1,39 +1,38 @@
 <section class="my-4">
-  {{ if .Envelope }}
   <section>
     <div class="mb-4 md:mb-0">
-      <span class="font-semibold">Transaction ID: {{ .Envelope.Transaction.Txid }}</span>
+      <span class="font-semibold">Transaction ID: {{ .Transaction.Txid }}</span>
```

Comment:
The `Txid` is included in the response, but we may need the `TransactionID` here.

Reply:
Can you remind me what the difference is between those two?

Reply:
The `transaction ID` is generated by the originator and it must be a UUID. The `Txid` is an ID that's unique to the blockchain or network.